hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/codec/EncodingFacade.java

```diff
@@ -204,7 +248,12 @@ protected enum FunctionType {
         ALLOWANCE,
         APPROVE,
         GET_APPROVED,
-        IS_APPROVED_FOR_ALL
+        IS_APPROVED_FOR_ALL,
+        HAPI_ALLOWANCE,
```

Comment:
Q: Does this imply previous `Approve`/`Allowance` `FunctionTypes` are of type ERC?
If so should we be explicit with an `ERC_` prefix like you've used in other cases?

Reply:
Updated all names